core/src/Platforms/iOS/EmbeddedWebview/WKWebNavigationDelegate.cs

```diff
@@ -73,7 +73,8 @@ public override void DecidePolicy(WKWebView webView, WKNavigationAction navigati
                    requestUrlString.StartsWith(BrokerConstants.BrowserExtInstallPrefix, StringComparison.OrdinalIgnoreCase))
             {
                 AuthenticationAgentUIViewController.callbackMethod(new AuthorizationResult(AuthorizationStatus.Success, requestUrlString));
```

Comment:
>AuthenticationAgentUIViewController.callbackMethod(new AuthorizationResult(AuthorizationStatus.Success, requestUrlString)); [](start = 16, length = 123)

remove ?

Reply:
done...thanks. Forgot to remove that line.